gensim/models/word2vec.py

```diff
@@ -100,15 +101,16 @@
 from types import GeneratorType
 
 logger = logging.getLogger(__name__)
-
+logging.basicConfig(format='%(asctime)s : %(levelname)s : %(message)s')
```

Comment:
Not sure if this should be present here, logging config is usually left up to either `__main__` or someone using the module.